lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -7,7 +7,14 @@
    calculator.ceremony_country == 'monaco' ||
    calculator.ceremony_country == 'france' ||
    calculator.ceremony_country == 'new-caledonia' ||
-   calculator.ceremony_country == 'wallis-and-futuna' %>
+   calculator.ceremony_country == 'wallis-and-futuna' ||
+   calculator.ceremony_country == 'french-guiana' ||
+   calculator.ceremony_country == 'french-polynesia' ||
+   calculator.ceremony_country == 'guadeloupe' ||
+   calculator.ceremony_country == 'martinique' ||
+   calculator.ceremony_country == 'mayotte' ||
+   calculator.ceremony_country == 'reunion' ||
+   calculator.ceremony_country == 'st-pierre-and-miquelon' %>
```

Comment:
Minor: As per my previous comment, I wonder whether it's worth trying to capture the concept of "France and its overseas territories" somehow. It feels very slightly as if we're losing this concept.